dcm4che-core/src/main/java/org/dcm4che3/io/DicomInputStream.java

```diff
@@ -83,9 +84,9 @@ public enum IncludeBulkData { NO, YES, URI }
         LoggerFactory.getLogger(DicomInputStream.class);
 
     private static final String UNEXPECTED_NON_ZERO_ITEM_LENGTH =
-        "Unexpected item value of {} #{} @ {}";
+        "Unexpected item value of {} #{} @ {} during {}";
```

Comment:
I would suggest opening these improvements in the log messages also as a PR for master.

Reply:
OK, I will create a pull request for this.